### PR TITLE
8 make readme more clear and user friendly

### LINKED ISSUE #8: Make README more clear and user-friendly
The [current README](https://github.com/VOLTTRON/copier-poetry-volttron-agent) should be improved so that a user can quickly get started. Right now, there are issues installing the required dependency 'copier' which need to included in the README.   See https://github.com/copier-org/copier/issues/574

At the bare minimum the README should include:

* A high level description of the customer/user workflow; it should set expectations on what the user will get after using this tool
* A link to an Agent example repo of using this tool
* Prerequisites section with scripts and/or links to install any required software
* Documentation on configuring and using Github Workflows
* A Quick-start section for users who don't want to read the README and just want to start coding
* Recommended ways of using this tool (i.e. Docker environment, virtual machine, bare metal)

### PR DESCRIPTION
**Pull request by bonicim**
Fixes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,43 +7,71 @@ Many thanks to the project [copier poetry](https://github.com/pawamoy/copier-poe
 for starting this project and allowing the reuse of the codebase.  Without this
 starting point the agent templates would not be what they are today!
```
Comment (kefeimo) [inline README.md:8]: The copy-and-paste is incomplete.

Many thanks to the project [copier poetry](https://github.com/pawamoy/copier-poetry)
for starting this project and allowing the reuse of the codebase.  Without this
starting point the agent templates would not be what they are today!

**Review by kefeimo — CHANGES_REQUESTED** [review]

**Comment by schandrika** [comment]
After the workflow diagram, could you put in the steps in words. 
ie. 
1. Modify agent code
2. git init
3. git commit (mention this run the pre commit check)
4. fix issue
5. Add remote
6. git push
7. When they are ready how to push to pypi ?

File: README.md (hunk 1 above)
Comment (bonicim) [reply README.md:8]: I don't understand. What part is incomplete? Can you elaborate please?

**Comment by bonicim** [comment]
> After the workflow diagram, could you put in the steps in words. ie.
> 
> 1. Modify agent code
> 2. git init
> 3. git commit (mention this run the pre commit check)
> 4. fix issue
> 5. Add remote
> 6. git push
> 7. When they are ready how to push to pypi ?

Will do.

File: README.md (hunk 1 above)
Comment (kefeimo) [reply README.md:8]: I missed the part that is folded. My bad. Nothing wrong here.

**Review by kefeimo — APPROVED** [review]

**Pull request merged**